client/my-sites/upgrades/domain-management/edit/registered-domain.jsx

```diff
@@ -58,8 +58,9 @@ const RegisteredDomain = React.createClass( {
 			transferPath = paths.domainManagementTransfer( slug, name );
 
 		if ( hasPrivacyProtection ) {
+			let label;
 			if ( privateDomain ) {
-				return this.getLabel( {
+				label = this.getLabel( {
```

Comment:
Ah, didn't figure. Thanks!

On Tue, Mar 22, 2016 at 10:35 PM Igor Klimer <notifications@github.com>
wrote:

> In client/my-sites/upgrades/domain-management/edit/registered-domain.jsx
> <https://github.com/Automattic/wp-calypso/pull/4194#discussion_r57073552>:
>
> >  			if ( privateDomain ) {
> > -				return this.getLabel( {
> > +				label = this.getLabel( {
>
> That's why I mentioned the eslint rule and linked to the documentation for
> it :) The solution is to just use this format:
>
> if (stuff) {
>    return foo;
> }
> return bar;
>
> instead of
>
> if (stuff) {
>    return foo;
> } else {
>    return bar;
> }
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Automattic/wp-calypso/pull/4194/files/a92fa9c8c3dc81f63d9d3b2074489fa2a3767d23#r57073552>
>